app/forms/nsm/steps/disbursement_cost_form.rb

```diff
@@ -23,11 +23,11 @@ def calculation_rows
         [
           [translate(:before_vat), translate(:after_vat)],
           [{
-            text: NumberTo.pounds(total_cost_pre_vat),
+            text: NumberTo.pounds(total_cost_pre_vat.to_f),
```

Comment:
This is still changing the behaviour slightly. When you first load the form page for an "other" disbursement type, total_cost_without_vat is nil, so previously `total_cost_pre_vat` would be nil and so we'd pass in nil to `NumberTo` and that would result in the UI showing "£". Now, because you're calling `.to_f`, when `total_cost_pre_vat` is nil, we end up passing 0 (`nil.to_f`) to NumberTo, so what shows up in the UI is "£0.00". 

If you want to keep it the same you'd want something like `NumberTo.pounds(total_cost_pre_vat unless total_cost_pre_vat.is_a?(String)`

Reply:
added a method that does this